tests/php/test-tag-and-attribute-sanitizer.php

```diff
@@ -1410,6 +1424,9 @@ static function() {
 							<amp-list-load-more load-more-end>
 								Congratulations! You reached the end.
 							</amp-list-load-more>
+							<div fetch-error align="center">
```

Comment:
If this is added here for testing, shouldn't the fallback and placeholder divs be in here too?